spec/helpers/blog_helper_spec.rb

```diff
@@ -23,19 +23,19 @@
         ['2017 ', ' 2017 '],
         ['foo-bar', ' foo --  bar']
       ].each do |arr|
-        fake_post = OpenStruct.new(title: arr[1], wordpress_url: '')
+        fake_post = OpenStruct.new(title: arr[1])
         expect(arr[0]).not_to eq(BlogHelper::url_slug(fake_post))
       end
     end
 
-    it "should return the correct url when a wordpress_url is provided" do
-      @posts.select{ |post| post.wordpress_url&.present? }.each do |post|
-        expect(BlogHelper::url_slug(post)).to eq(BlogHelper::url_slug_from_wordpress_url(post[:wordpress_url]))
+    it 'should return the correct url when a url slug is provided' do
+      @posts.select { |post| post.url_slug.present? }.each do |post|
+        expect(BlogHelper::url_slug(post)).to eq(post.url_slug)
       end
     end
```

Comment:
LGTM…but maybe it's worth adding a `expect(@posts.length).to.be.greater.than(…)` or `expect.assertions(42)` sort of check?  Not sure the syntax in rspec, but otherwise this might just be an empty test?